distributed/deploy/adaptive.py

```diff
@@ -20,6 +20,11 @@ class Adaptive(object):
     scheduler: distributed.Scheduler
     cluster: object
         Must have scale_up and scale_down methods/coroutines
+    startup_cost : int, default 1
+        Factor representing how costly it is to start an additional worker.
+        Affects quickly to adapt to high tasks per worker loads
+    scale_factor : int, default 2
```

Comment:
Earlier I had this as a class attribute, but I moved it to be an argument to `__init__` to be more in line with `startup_cost` (it's hard coded to 2 on master)